pallets/proof-of-stake/src/tests.rs

```diff
@@ -142,6 +148,9 @@ fn liquidity_rewards_three_users_burn_W() {
 		TokensOf::<Test>::create(&acc_id, amount).unwrap();
 		TokensOf::<Test>::create(&acc_id, 10000).unwrap();
 
+		let is_liquidity_token_mock = MockValuationApi::is_liquidity_token_context();
```

Comment:
consider putting mock into `execute_with_default_mocks` in mocks.rs to avoid duplications